tests/extensions/test_timing.py

```diff
@@ -18,3 +18,5 @@ def test_timing():
          main_loop.log[iterations]['each_time_train_this_epoch']) / 2,
         main_loop.log.current_row['each_second_time_train_this_epoch'],
         atol=1e-2)
+    assert 'each_time_read_data_this_epoch' in main_loop.log[iterations]
```

Comment:
This is only for testing purposes. I used "each_time" prefix, so by default it would be "read_data_this_epoch", like before.